### DISCUSSION TITLE
Is arquero being maintained?

### DISCUSSION THREAD

**Discussion by oskbor** [uwdata/arquero#351] [General] [answered]
Hi, this seems like a very interesting and powerful library! But it also seems that it is not being updated recently which gives me pause. For example there have been no commits for 6 months and there are five open PRs on the repository. This can of course be due to the library being "done", but often things like updating the dependencies can be nice for consumers and a never ending chore.

For people using this library, would you recommend using it or is there something similar that you would recommend instead?

**Comment by jheer** [comment, marked as answer]
I am still maintaining this library and use it myself regularly. There is not much activity on the repo as the library is more-or-less "done", but I'm happy to address bugs and will periodically update dependencies, etc. I'm planning another update sometime in the coming month.

**Reply by oskbor** [reply]
Glad to hear that! 🙏👍
